app/src/main/java/co/nimblehq/compose/crypto/ui/screens/compose/composables/home/HomeScreen.kt

```diff
@@ -0,0 +1,70 @@
+package co.nimblehq.compose.crypto.ui.screens.compose.composables.home
```

Comment:
How do you think if we get rid of `composables` package? It's a bit redundant imo since we already have `compose` package. Or would you mind to explain your idea if we would have it? 😄

Reply:
I will remove it. We can create a sub-directory inside `compose` following the bottom bar(home, search, profile, ...) is enough.🙏